hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/components/tables/containersTable.tsx

```diff
@@ -0,0 +1,258 @@
+  },
+  {
+    title: 'No. of Keys',
+    dataIndex: 'keys',
+    key: 'keys',
+    sorter: (a: Container, b: Container) => a.keys - b.keys
+  },
+  {
+    title: 'Actual/Expected Replica(s)',
+    dataIndex: 'expectedReplicaCount',
+    key: 'expectedReplicaCount',
+    render: (expectedReplicaCount: number, record: Container) => {
+      const actualReplicaCount = record.actualReplicaCount;
+      return (
+        <span>
+          {actualReplicaCount} / {expectedReplicaCount}
+        </span>
+      );
+    }
+  },
+  {
+    title: 'Datanodes',
+    dataIndex: 'replicas',
+    key: 'replicas',
+    render: (replicas: ContainerReplicas[]) => {
+      const renderDatanodes = (replicas: ContainerReplicas[]) => {
+        return replicas?.map((replica: any, idx: number) => (
+          <div key={idx} className='datanode-container-v2'>
+            <NodeIndexOutlined /> {replica.datanodeHost}
+          </div>
+        ))
+      }
+
+      return (
+        <Popover
+          content={renderDatanodes(replicas)}
+          title='Datanodes'
+          placement='bottomRight'
+          trigger='hover'>
+          <strong>{replicas.length}</strong> datanodes
+        </Popover>
+      )
+    }
+  },
+  {
+    title: 'Pipeline ID',
+    dataIndex: 'pipelineID',
+    key: 'pipelineID'
+  },
+  {
+    title: 'Unhealthy Since',
+    dataIndex: 'unhealthySince',
+    key: 'unhealthySince',
+    render: (unhealthySince: number) => getFormattedTime(unhealthySince, 'lll'),
+    sorter: (a: Container, b: Container) => a.unhealthySince - b.unhealthySince
+  }
+];
+
+const KEY_TABLE_COLUMNS: ColumnsType<KeyResponse> = [
+  {
+    title: 'Volume',
+    dataIndex: 'Volume',
+    key: 'Volume'
+  },
+  {
+    title: 'Bucket',
+    dataIndex: 'Bucket',
+    key: 'Bucket'
+  },
+  {
+    title: 'Key',
+    dataIndex: 'Key',
+    key: 'Key'
+  },
+  {
+    title: 'Size',
+    dataIndex: 'DataSize',
+    key: 'DataSize',
+    render: (dataSize: number) => <div>{size(dataSize)}</div>
+  },
+  {
+    title: 'Date Created',
+    dataIndex: 'CreationTime',
+    key: 'CreationTime',
+    render: (date: string) => getFormattedTime(date, 'lll')
+  },
+  {
+    title: 'Date Modified',
+    dataIndex: 'ModificationTime',
+    key: 'ModificationTime',
+    render: (date: string) => getFormattedTime(date, 'lll')
+  }
+];
+
+const ContainerTable: React.FC<ContainerTableProps> = ({
+  data,
+  loading,
+  selectedColumns,
+  expandedRow,
+  expandedRowSetter,
+  searchColumn = 'containerID',
+  searchTerm = ''
+}) => {
+
+  const cancelSignal = useRef<AbortController>();
+
+  function filterSelectedColumns() {
+    const columnKeys = selectedColumns.map((column) => column.value);
+    return COLUMNS.filter(
+      (column) => columnKeys.indexOf(column.key as string) >= 0
+    );
+  }
+
+  function loadRowData(containerID: number) {
+    const { request, controller } = AxiosGetHelper(
+      `/api/v1/containers/${containerID}/keys`,
```

Comment:
In the current scenario, I have a container with an ID of 1 that contains 69 keys. When I click the plus icon to view the associated keys, I can see only 10 keys per page. Given that there are 69 keys in total, the pagination limits the view to just the first 50 keys. As a result, I am unable to access the keys beyond the 50th entry.

<img width="1660" alt="image" src="https://github.com/user-attachments/assets/06bfc5b4-4d82-45c5-ba07-3e715a3aed2e">